src/definitions/modules/dropdown.js

```diff
@@ -3557,13 +3557,13 @@
                     },
 
                     // https://github.com/fomantic/Fomantic-UI/issues/2782
-                    // https://jsfiddle.net/wdyjfvz0/
-                    assumeUnescapedAmpersand: function (string) {
+                    // https://jsfiddle.net/3efL7jnt/
+                    assumeUnescapedAmpLtGt: function (string) {
                         if (settings.preserveHTML) {
                             return string;
                         }
 
-                        return string.replace('&amp;', '&');
+                        return string.replaceAll('&amp;', '&').replaceAll('&lt;', '<').replaceAll('&gt;', '>');
```

Comment:
Using replaceAll will force us to limit the next Version to iOS13.1+
I tend to do that as i also think about other newer syntax elements which will make the codebase simpler (we officially support the last 4 iOs Version, so that would be fine anyway) 

Otherwise the current codebase will work on iOS 10+

Just saying

Reply:
fixed by coincidence, see https://github.com/fomantic/Fomantic-UI/pull/3231#discussion_r2015020379